CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ endif()
 # Set project name
 project (storm
 		LANGUAGES CXX C
-		VERSION 1.10.0.1)
+		VERSION 1.11.0)
```

Comment:
In the future, maybe we can add to the CI workflow that a commit that changes this back to x.y.z.1.

Reply:
You mean immediately after a release we increase it by 1 indicate the next development version? That would indeed be a good idea.